packages/react/test/specs/components/Dropdown/test-utils.tsx

```diff
@@ -0,0 +1,105 @@
+import * as React from 'react'
+import * as _ from 'lodash'
+
+import Dropdown, { DropdownProps } from 'src/components/Dropdown/Dropdown'
+import DropdownSearchInput from 'src/components/Dropdown/DropdownSearchInput'
+import { findIntrinsicElement, mountWithProvider } from 'test/utils'
+
+const items = ['item0', 'item1', 'item2', 'item3', 'item4', 'item5']
+
+const renderDropdown = (props: DropdownProps = {}) => {
+  const wrapper = mountWithProvider(<Dropdown items={items} {...props} />)
+  const triggerButtonWrapper = findIntrinsicElement(
+    wrapper,
+    `.${Dropdown.slotClassNames.triggerButton}`,
+  )
+  const toggleIndicatorWrapper = findIntrinsicElement(
+    wrapper,
+    `.${Dropdown.slotClassNames.toggleIndicator}`,
+  )
+  const searchInputWrapper = findIntrinsicElement(
+    wrapper,
+    `.${DropdownSearchInput.slotClassNames.input}`,
+  )
+  const itemsListWrapper = findIntrinsicElement(wrapper, `.${Dropdown.slotClassNames.itemsList}`)
+  const getItemsWrapper = () => findIntrinsicElement(wrapper, `.${Dropdown.slotClassNames.item}`)
+  const getSelectedItemsWrapper = () =>
+    findIntrinsicElement(wrapper, `.${Dropdown.slotClassNames.selectedItem}`)
+  const getSelectedItemWrapperAtIndex = index => getSelectedItemsWrapper().at(index)
+  const getItemWrapperAtIndex = index => getItemsWrapper().at(index)
+
+  return {
+    wrapper,
+    triggerButton: triggerButtonWrapper.length ? triggerButtonWrapper.getDOMNode() : null,
+    toggleIndicator: toggleIndicatorWrapper.length ? toggleIndicatorWrapper.getDOMNode() : null,
+    itemsList: itemsListWrapper.getDOMNode(),
+    searchInput: searchInputWrapper.length
+      ? searchInputWrapper.getDOMNode<HTMLInputElement>()
+      : null,
+    getA11yMessageContainer: () => findIntrinsicElement(wrapper, '[role="status"]').getDOMNode(),
+    getItems: () => getItemsWrapper().map(nodeWrapper => nodeWrapper.getDOMNode()),
+    getSelectedItems: () => getSelectedItemsWrapper().map(nodeWrapper => nodeWrapper.getDOMNode()),
+    getItemAtIndex: index => getItemWrapperAtIndex(index).getDOMNode(),
+    getSelectedItemAtIndex: index => getSelectedItemWrapperAtIndex(index).getDOMNode(),
```

Comment:
**(!!)** I confused with naming there as `getItemAtIndex` doesn't say to me that I will receive DOM node there

Reply:
Why not? Everything returns DOM nodes and the ones ending with `Wrapper` return the wrapper.

Reply:
`getItemNodeAtIndex`?